lib/locales/de.js

```diff
@@ -11,7 +11,68 @@
 module.exports = {
     moduleType: 'locale',
     name: 'de',
-    dictionary: {},
+    dictionary: {
+        'Autoscale': 'Automatische Skalierung',                                              // components/modebar/buttons.js:139
+        'Box Select': 'Box Auswahl',                                            // components/modebar/buttons.js:103
+        'Click to enter Colorscale title': 'Klicken, um den Farbskalatitel einzugeben',                       // plots/plots.js:437
```

Comment:
I wonder if 'click to' and thus 'Klicken, um' is really needed. 

In HTML you should never use a hyperlink named "here" like "[click here](www.google.com) to open Google". It is always preferable to use "[Open Google](www.google.com)". I feel like help texts in tooltips, etc should act the same.

Same for related translations. What do you think?

Reply:
Our rationale was that it’s not obvious that this is a clickable link at all, unlike a specially-styled item within text - and we purposely make this placeholder text lighter than other text, not more prominent like links usually are, so it doesn’t distract you too much if you don’t want a title at all.

Reply:
> I wonder if 'click to' and thus 'Klicken, um' is really needed.

FYI, I thought it best to leave this in here, as @JonasGutermuth did an admirable job keeping the translation short. But @jnehlmeier it's true that this may not be completely necessary. We've been discussing this in other contexts, and if the translation is much longer than the English and users can be expected to understand that this is a clickable link without the "Click to", you're right it may be better to omit it so it doesn't take up too much space - see https://github.com/plotly/plotly.js/pull/2252#discussion_r161635675

Reply:
Good design is to never tell a user that something is clickable, it must visually be clear that it is clickable. Desaturated, light text might look good, but gives poor visual clues. 

So maybe you should think about restyling the elements in question. E.g. make everything clickable the same color, or add a consistent edit icon, change the mouse cursor on hover, or even remove clickable placeholder text and instead use input boxes in some edit panel.

Reply:
(Not to beat a dead horse, and I don't mean to say we would never change this design, @jnehlmeier raises good points, just hoping this conversation may be useful to future translation contributors)

> Good design is to never tell a user that something is clickable

Definitely a good rule of thumb, though a similar thing happens in Powerpoint and Keynote:
<img width="1273" alt="screen shot 2018-01-16 at 6 55 54 am" src="https://user-images.githubusercontent.com/2678795/34987946-f6c85bda-fa8a-11e7-8d03-a502412d17a2.png">

In our [Chart editor](plot.ly/create) (which is the original context for which `editable: true` was created, though of course we hope it finds other uses) we *do* have input boxes in an edit panel as an option, and these placeholders *do* get an edit cursor on hover (which titles do not get if they are not editable). Our design goal was to make it clear that you *can* just click and type, but in a soft enough way that users won't feel the need to delete placeholders for titles they don't want, and they will understand that, just like in PPT/Keynote, when you're done editing and view your plot in a non-editable context these placeholders disappear if not replaced by other text.